src/main/java/org/ibp/api/rest/study/StudyEntryResource.java

```diff
@@ -158,4 +161,13 @@ public long getCount() {
 
 		return new ResponseEntity<>(this.studyEntryService.getStudyEntriesMetadata(studyId, programUUID), HttpStatus.OK);
 	}
+
+	@ApiOperation(value = "Get existing Germplasm ids", notes = "Retrieves existing Germplasm ids in the study from the given gid list")
+	@PreAuthorize("hasAnyAuthority('ADMIN','STUDIES','MANAGE_STUDIES')")
+	@RequestMapping(value = "/{crop}/programs/{programUUID}/studies/{studyId}/entries/existing-gids/{gids}", method = RequestMethod.GET)
```

Comment:
As discussed, check if you can resolve the existing GIDs from the frontend by calling existing get study entries